### PR TITLE
Add new Custom Rector Rule to Pass true to 3rd parameter of in_array when no value provided

### PR DESCRIPTION
**Pull request by samsonasik**
Added new Custom Rector Rule to Pass `true` to 3rd parameter of `in_array` when no value provided.

**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

**Comment by samsonasik** [comment]
All green :tada:

**Review by paulbalandan — APPROVED** [review]
Stricter means less prone to errors. 🥳

Can we have also, maybe in a separate PR, other functions which have a `$strict` parameter also set to true, like in `base64_decode` and other functions?

**Comment by samsonasik** [comment]
@paulbalandan Yes, next, I can create new rule for other functions, or refactor it to accept register new function with expected arg position to be true.

File: utils/Rector/PassStrictParameterToFunctionParameterRector.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+/**
+ * Pass strict to function parameter on specific position argument when no value provided
+ */
+final class PassStrictParameterToFunctionParameterRector extends AbstractRector
+{
+	private const FUNCTION_WITH_ARG_POSITION = [
+		// position start from 0
+		'in_array' => 2,
+	];
```
Comment (samsonasik) [inline utils/Rector/PassStrictParameterToFunctionParameterRector.php:24]: @paulbalandan I've refactored to allow register more functions to specific position to be filled with strict parameter in the above constant.

**Comment by samsonasik** [comment]
latest phpstan release seems cause an error https://github.com/phpstan/phpstan/releases/tag/0.12.51

**Comment by samsonasik** [comment]
I currently ignore new unrelated phpstan error due new release of phpstan

**Comment by samsonasik** [comment]
All green 🎉

File: utils/Rector/PassStrictParameterToFunctionParameterRector.php (hunk 1 above)
Comment (paulbalandan) [reply utils/Rector/PassStrictParameterToFunctionParameterRector.php:24]: Thank you for indulging my request that fast.

**Review by michalsn — APPROVED** [review]

**Comment by samsonasik** [comment]
@paulbalandan @michalsn Thank you for the review.

**Pull request merged**